lib/fedex_rest_api/address.rb

```diff
@@ -0,0 +1,35 @@
+require "fedex_rest_api/base"
+
+class FedexRestApi::Address
+  attr_reader :address_object
+
+  ADDRESS_VALIDATION_URL = "/address/v1/addresses/resolve".freeze
+
+  def initialize(address_object)
+    @address_object = address_object
+  end
+
+  def validate
+    response = HTTParty.post(env_url,
+      headers: {
+        "Content-Type" => 'application/json',
+        "Authorization" => "Bearer #{address_object[:access_token]}"
+      },
+      body: {
+        "addressesToValidate": address_object[:addresses]
+      }.to_json
+    )
+
+    raise FedexRestApi::ApiError if response['errors'] || response['error_description']
+    response
+  end
+
+  def env_url
+    if address_object[:environment] == FedexRestApi::Base::PRODUCTION_ENV
+      FedexRestApi::Base::PRODUCTION_ADDRESS_VALIDATION_URL
```

Comment:
I'm struggling to see what this line is meant to do. As far as I can tell the next line will be the return value.

Reply:
Ope! Good catch. That was a remnant left over from my refactor!